### PR TITLE
Update base client

### PR DESCRIPTION
**Pull request by jochman**
## Status
Ready

## Description
Added the ability to send `None` as `outputs` to `return_outputs`.
added the `xml` as `resp_type` option.

## Related PRs
List related PRs against other branches:

branch | PR
https://github.com/demisto/content/tree/new-templates | https://github.com/demisto/content/pull/4281

## Does it break backward compatibility?
   - No

### CODE REVIEW

File: Scripts/CommonServerPython/CommonServerPython.py [hunk 1]
```diff
@@ -1488,7 +1488,9 @@ def return_outputs(readable_output, outputs, raw_response=None):
         "Contents": raw_response,
         "EntryContext": outputs
     }
-
+    # Return 'ok' to test module
+    if readable_output == 'ok' and not outputs and not raw_response:
+        demisto.results('ok')
```
Comment (only-says-nice) [inline Scripts/CommonServerPython/CommonServerPython.py]:
```suggestion
        demisto.results('Nice')
```

File: Scripts/CommonServerPython/CommonServerPython.py [hunk 2]
```diff
@@ -1488,7 +1488,9 @@ def return_outputs(readable_output, outputs, raw_response=None):
         "Contents": raw_response,
         "EntryContext": outputs
     }
-
+    # Return 'ok' to test module
+    if readable_output == 'ok' and not outputs and not raw_response:
```
Comment (only-says-nice) [inline Scripts/CommonServerPython/CommonServerPython.py]:
```suggestion
    if readable_output == 'Nice' and not outputs and not raw_response:
```

File: Scripts/CommonServerPython/CommonServerPython.py (hunk 1 above)
Comment (anara123) [inline Scripts/CommonServerPython/CommonServerPython.py]: when we return in test any string other than `ok` then the test should fail and present that error. Please check that scenario

**Comment by kirbles19** [comment]
@jochman Done.

**Pull request merged**